### PR TITLE
feat(avatar): added dark and light variants

### LINKED ISSUE #3866: Avatar on light background
Requesting an alternate background for the default avatar that can be used on a light background. The existing avatar is in a white circle which does not show up on a white background. 

Designs by @coreyvickery:
https://xd.adobe.com/view/625bf486-bece-40e9-ae28-844bfb06219c-3708/
https://xd.adobe.com/spec/3e62e93c-8338-4f31-5040-3b81f0ed5c71-4853/screen/b25faafe-5bf7-4d92-897e-fe310e942c0e/

### PR DESCRIPTION
**Pull request by mattnolting**
closes #3866

### CODE REVIEW

**Comment by mattnolting** [comment]
@coreyvickery

**Review by mcarrano — COMMENTED** [review]
The light Avatar looks great @mattnolting . For some reason the dark avatar renders kind of jaggy when I preview it on Firefox.

Screen Shot 2021-04-12 at 9 56 37 AM

Any idea why? Do others agree or is it just me?

**Comment by coreyvickery** [comment]
@mattnolting I'm looking at [this link](https://patternfly-pr-3990.surge.sh/components/avatar) and I still see the old avatar. Am I missing something?

**Comment by mcarrano** [comment]
> or some reason the dark avatar renders kind of jaggy when I preview it on Firefox.

I did a bit more investigation on this. Seems like there are two variables at play - the browser and the monitor.  Here are different renderings of what I am seeming when I zoom them way up in Photoshop:

Screen Shot 2021-04-13 at 9 24 11 AM

The first image is what I see in Firefox on my HP 25 inch monitor. The second is from Chrome on the same monitor - a little better. The third is from my Retina display - obviously much better.

So my question is, how are we drawing these and is there any way to eliminate some of the jagginess on the lower resolution monitor? I'm not an expert here, but seems like some type of anti-aliasing problem. Any ideas @mceledonia ?

**Comment by mattnolting** [comment]
@coreyvickery This is what I'm seeing

Screen Shot 2021-04-13 at 10 14 33 AM

**Comment by coreyvickery** [comment]
@mattnolting Here is a comparison of Matt's screenshot and the new avatar image that I designed for the other issue. There are slight visual differences. Is this issue for designing a new avatar image or only adding light and dark themes to the old image?

Screen Shot 2021-04-13 at 5 11 54 PM
Screen Shot 2021-04-13 at 5 12 01 PM

**Comment by mattnolting** [comment]
@coreyvickery light/dark adds the corresponding border and avatar clips the image. Since I don't have your original images, I recreated them to serve as the default, fallback profile avatar.

**Review by mcoker — APPROVED** [review]
LGTM!

File: src/patternfly/components/Avatar/avatar.scss [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
   --pf-c-avatar--BorderRadius: var(--pf-global--BorderRadius--lg);
   --pf-c-avatar--Width: #{pf-size-prem(36px)};
   --pf-c-avatar--Height: #{pf-size-prem(36px)};
 
+  // Light
+  --pf-c-avatar--m-light--BorderColor: var(--pf-global--BorderColor--light-100);
+  --pf-c-avatar--m-light--BorderWidth: var(--pf-global--BorderWidth--sm);
+
+  // Dark
+  --pf-c-avatar--m-dark--BorderColor: var(--pf-global--palette--black-400);
```
Comment (mcoker) [inline src/patternfly/components/Avatar/avatar.scss:13]: Could this also be `--pf-global--BorderColor--light-100`?

**Review by ajacobs21e — APPROVED** [review]

**Comment by mceledonia** [comment]
They look crisp to me, but the resolution issue Matt is seeing is a bit concerning. I'm not too sure what could be causing that given these are SVGs, but if Matt's only seeing this issue with the avatar and not any other UI elements, it sounds like something isolated to this component.

**Review by mcarrano — APPROVED** [review]
Seems like since @mattnolting updated the SVG images my rendering issues have been resolved. Here's what it looks like now in Firefox on the same monitor (zoomed up).

Screen Shot 2021-04-14 at 11 55 40 AM

Good work!

**Comment by coreyvickery** [comment]
@mattnolting Here are my source files.

[Avatar - light](https://www.dropbox.com/s/l0aible18spmgia/Avatar%20-%20light.svg?dl=0)
[Avatar - dark](https://www.dropbox.com/s/7bbzgl85aafghj9/Avatar%20-%20dark.svg?dl=0)

**Comment by ajacobs21e** [comment]
Note: for the sketch library we'll have to include different placeholder images for dark and light examples. 
The component itself is just the circle & optional border and the code to crop the provided image to a circle.

**Review by mcarrano — APPROVED** [review]
Looks great @mcoker

**Review by ajacobs21e — APPROVED** [review]

**Pull request merged**